src/Analysis/GWASApp/GWASApp.css

```diff
@@ -0,0 +1,260 @@
+.GWASApp .steps-content {
```

Comment:
When do we call something `GWASApp` vs `GWASUI`?

Reply:
```GWASUI``` is a class name ported over from the initial version of the stylesheet: when we created the new version the app, we copied the style sheet  and much of the markup associated with the first version of the application. 

GWASApp is a class name introduced per your suggestion for this ticket: all instances of the class name ```GWASV2``` have been changed to ```GWASApp```